package.json

```diff
@@ -5,7 +5,7 @@
     "body-parser": "1.15.0",
     "cors": "2.7.1",
     "express": "4.13.4",
-    "fh-mbaas-api": "5.11.2",
+    "fh-mbaas-api": "5.12.14",
```

Comment:
the latest, greatest, is `5.13.1`, which is Apache 2.0 license - before, there was no OSS license